### DISCUSSION TITLE
How to use Audit Trail Module ?

### DISCUSSION THREAD

**Discussion by EricBui619** [OrchardCMS/OrchardCore#9233] [Q&A]
My current project is using Orchard Core as the backend and my client require a feature that allow them to see the history of all versions of content items. Also, they are able to compare and see the change among versions. I found Audit Trail Module that has enough features  we want. But this module is being in development and it really doesn't make sense to integrate all codebase of Audit module to my project.

So currently, Is there any possible way for me to implement my client's requirements based on OrchadCore?

**Comment by Skrypt** [comment]
Strongly suggest waiting on the Audit Trail module. Else, yes you can create your own.

**Reply by EricBui619** [reply]
@Skrypt Thanks for your suggestion. I saw a demo video of Lombiq about Audit Trail module 4 months ago and it looked so good but I wonder when is official Audit Trail module going to be released?

**Comment by hishamco** [comment]
@EricArita you could play with the PR #6679 until it's officially merged

**Reply by EricBui619** [reply]
Thank u, i will try it
